test-frontend.sh

```diff
@@ -12,10 +12,18 @@ if [ "$OPENSHIFT_CI" = true ]; then
     JEST_SUITE_NAME="OpenShift Console Unit Tests" JEST_JUNIT_OUTPUT_DIR="$ARTIFACT_DIR" yarn run test --ci --reporters=default --reporters=jest-junit
 else
     yarn run test
+yarn i18n
```

Comment:
Move this outside of the if/else. It needs to happen in CI, too. I've move it just before the `GIT_STATUS` check below.